tests/blanket-options.js

```diff
@@ -11,7 +11,7 @@ var options = {
     autostart: true,
     lcovOptions: {
       outputFile: 'coverage/lcov.info',
-      renamer (fileName) {
+      renamer: function (fileName) {
```

Comment:
I randomly saw a test fail where it complained the `:` was missing here. I have no idea why a test would say that but did this for posterities sake.